src/widgets/forms.js

```diff
@@ -155,29 +181,149 @@ forms.field[UI.ns.ui('Options').uri] = function (
+/**          Multiple field: zero or more similar subFields
 **
+** @param {Document} dom The HTML Document object aka Document Object Model
+** @param {Element?} container  If present, the created widget will be appended to this
+** @param {Map} already A hash table of (form, suject) kept to prevent recusive forms looping
+** @param {Node} subject The thing about which the form displays/edits data
+** @param {Node} form The form or field to be rendered
+** @param {Node} store The web document in which the data is
+** @param {function(ok, errorMessage)} callbackFunction Called when data is changed?
+**
+** @returns {Element} The HTML widget created
 */
 forms.field[UI.ns.ui('Multiple').uri] = function (
   dom, container, already, subject, form, store, callbackFunction) {
-  // var plusIcon = UI.icons.originalIconBase + 'tango/22-list-add.png' // blue plus
+  /** Add an item to the control
+   * @param {Event} anyEvent if used as an event handler
+   * @param {Node} object The RDF object to be represented by this item.
+  */
+  function addItem (object) { // can be used as a event handler
+    async function deleteItem () {
+      if (ordered) {
+        for (let i = 0; i < list.elements.length; i++) {
+          if (list.elements[i].sameTerm(object)) {
+            list.elements.splice(i, 1)
+            saveListThenRefresh()
+            return
+          }
+        }
+      } else { // unordered
+        if (kb.holds(subject, property, object)) {
+          var del = [$rdf.st(subject, property, object, store)]
+          kb.updater.update(del, [], function (uri, ok, message) {
+            if (ok) {
+              body.removeChild(subField)
+            } else {
+              body.appendChild(error.errorMessageBlock(dom, 'Multiple: delete failed: ' + message))
+            }
+          })
+        }
+      }
+    }
+
+    /** Move the object up or down in the ordered list
+     * @param {Event} anyEvent if used as an event handler
+     * @param {Boolean} upwards Move this up (true) or down (false).
+    */
+    async function move (event, upwards) { // @@ possibly, allow shift+click to do move to top or bottom?
+      for (var i = 0; i < list.elements.length; i++) { // Find object in array
+        if (list.elements[i].sameTerm(object)) {
+          break
+        }
+      }
+      if (i === list.elements.length) {
+        alert('list move: not found element for ' + object)
+      }
+      if (upwards) {
+        if (i === 0) {
+          alert('@@ boop - already at top   -temp message')  // @@ make boop sound
+          return
+        }
+        list.elements.splice(i - 1, 2, list.elements[i], list.elements[i - 1])
+      } else { // downwards
+        if (i === list.elements.length - 1) {
+          alert('@@ boop - already at bottom   -temp message')   // @@ make boop sound
+          return
+        }
+        list.elements.splice(i, 2, list.elements[i + 1], list.elements[i])
+      }
+      saveListThenRefresh()
```

Comment:
You might want to put an `await` in front of the `saveListThenRefresh`? Otherwise, why bother having the `move` function be async at all?